tests/test_compositing.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from moviepy.editor import *
+
+def test_clips_array():
+    _red = ColorClip((1024,800), col=(255,0,0))
```

Comment:
@earney https://shahriar.svbtle.com/underscores-in-python#single-underscore-before-a-name-eg-code-class_1